### PR TITLE
fix: Update exports in package manifests

### PR DESCRIPTION
**Pull request by dmaretskyi**
### Motivation / Background

This Pull Request ...

### CODE REVIEW

File: packages/apps/plugins/plugin-attention/package.json [hunk 1]
```diff
@@ -9,15 +9,17 @@
...
+      },
+      "types": "./dist/types/src/index.d.ts"
     },
     "./meta": {
       "browser": "./dist/lib/browser/meta.mjs",
-      "import": "./dist/lib/browser/meta.mjs",
-      "require": "./dist/lib/node/meta.cjs",
-      "node": "./dist/lib/node/meta.cjs"
+      "node": {
+        "default": "./dist/lib/node/meta.cjs"
```
Comment (dmaretskyi) [inline packages/apps/plugins/plugin-attention/package.json:20]: The idea here is to eventually have separate "import" and "require" exports for ESM and CJS respectively

File: packages/common/crypto-names/package.json [hunk 2]
```diff
@@ -6,11 +6,19 @@
...
+      "browser": "./dist/lib/browser/index.mjs",
+      "node": {
+        "default": "./dist/lib/node/index.cjs"
+      },
+      "types": "./dist/types/src/index.d.ts"
+    }
+  },
+  "types": "dist/types/src/index.d.ts",
+  "typesVersions": {
+    "*": {}
```
Comment (wittjosiah) [inline packages/common/crypto-names/package.json:20]: seen this in a couple packages, is it necessary for something even though it's empty?

**Review by wittjosiah — APPROVED** [review]

File: packages/common/crypto-names/package.json (hunk 2 above)
Comment (dmaretskyi) [reply packages/common/crypto-names/package.json:20]: Not necessary but not harmful, I just didn’t want to complicate the tool code to handle this case

Comment (dmaretskyi) [reply packages/common/crypto-names/package.json:20]: If it’s too annoying I can add a case to remove it

**Pull request merged**